elfpy/time/time.py

```diff
@@ -33,18 +50,6 @@ def normalized_time(self):
             self.normalizing_constant,
         )
 
-    def __str__(self):
-        output_string = (
-            "StretchedTime(\n"
-            f"\t{self.days=},\n"
-            f"\t{self.normalized_time=},\n"
-            f"\t{self.stretched_time=},\n"
-            f"\t{self.time_stretch=},\n"
-            f"\t{self.normalizing_constant=},\n"
-            ")"
-        )
-        return output_string
-
 
 def current_datetime() -> datetime:
```

Comment:
i feel like we could remove all the datetime functions, convert from time to datetime in post-op if desired